modules/nodes-pods-autoscaling-creating-cpu.adoc

```diff
@@ -70,6 +70,53 @@ $ oc autoscale dc/<dc-name> \// <1>
 <2> Optionally, specify the minimum number of replicas when scaling down.
 <3> Specify the maximum number of replicas when scaling up.
 <4> Specify the target average CPU utilization over all the pods, represented as a percent of requested CPU. If not specified or negative, a default autoscaling policy is used.
++
+[TIP]
+====
+You can alternatively apply the following YAML to scale based on the percent of CPU:
```

Comment:
I'm a little jarred from seeing one openshift-kube-scheduler pod in the preceding output and then showing scaling of an app in this example.  Can you get the pod metrics for the sample app and replace the openshift-kube-scheduler output?